### ISSUE TITLE
Summary

### ISSUE THREAD

**Issue by jim-rosales** [mfb16/tea-cozy#5]
Grade: Meets Expectations

Summary:
Congrats on finishing the Tea Cozy project! You did an excellent job making your files readable by using descriptive class names and adding comments throughout your project. Apart from this, you did an excellent job correctly formatting and structuring your HTML file. 

One thing to note for future projects is formatting for your CSS files. Properly formatting your files enhances the readability of your code which promotes collaboration among team members and helps you while you are working on your project. Take a look at this [CSS](https://gist.github.com/jim-rosales/f7e0153c0bb6a2c9563d415b9b4d44dc) solution for the project and notice how it is formatted and how it differs to your code. 

After you have done this, you can read more about formatting CSS you from this [style guide](https://github.com/airbnb/css/network) which mentions best practices when writing CSS.

**Comment by mfb16** [comment]
Thank you for the review. Is it possible to send me the html file that goes with the CSS linked above? Im interested in seeing it.

**Comment by jim-rosales** [comment]
Yeah of course! Here is a [gist](https://gist.github.com/jim-rosales/46f2f87823e2546e3e1da4427e72bb48) of the HTML solution for the project.
